supervised/ensemble.py

```diff
@@ -73,6 +73,9 @@ def get_name(self):
     def get_metric_name(self):
         return self.metric.name
 
+    def get_metric(self):
+        return Metric({'name': self.get_metric_name()})
```

Comment:
No need to construct new object. The `self.metric` is already a `Metric` object.